src/DetailsView/components/interactive-header.tsx

```diff
@@ -25,13 +24,14 @@ export const InteractiveHeader = NamedFC<InteractiveHeaderProps>('InteractiveHea
         return <Header deps={props.deps} />;
     }
 
-    return (
-        <Header deps={props.deps}>
-            <Switcher deps={props.deps} pivotKey={props.selectedPivot} />
-            <GearOptionsButtonComponent
-                dropdownClickHandler={props.dropdownClickHandler}
-                featureFlags={props.featureFlagStoreData}
-            />
-        </Header>
+    const getNearItems = () => <Switcher deps={props.deps} pivotKey={props.selectedPivot} />;
```

Comment:
Update to match rename below?
```suggestion
    const getItems = () => <Switcher deps={props.deps} pivotKey={props.selectedPivot} />;
```

Reply:
updated